src/celengine/catentry.h

```diff
@@ -10,6 +10,13 @@ class UserCategory;
 class CatEntry
 {
 public:
+    enum DataDisposition
```

Comment:
`enum class DataDisposition` to enforce additional compiler's type safety checks